src/main/plugins/org.dita.pdf2/cfg/fo/attrs/loc-attr.xsl

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0'?>
+
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:fo="http://www.w3.org/1999/XSL/Format"
+                xmlns:rx="http://www.renderx.com/XSL/Extensions"
+                version="2.0">
+
+    <xsl:attribute-set name="releaseManagementTable">
+    </xsl:attribute-set>
+
+    <xsl:attribute-set name="releaseManagementTable.bookRelease.row">
```

Comment:
done
However the output looks quite boring now. Should I apply any other default attribute-sets?
![loc](https://cloud.githubusercontent.com/assets/3266567/15100702/7bc0e980-157a-11e6-86e0-ce0f9e41e32c.png)